COLLABORATOR_GUIDE.md

```diff
@@ -411,8 +411,11 @@ recommended but not required.
 
 ### Deprecations
 
-**Deprecation** refers to the identification of Public APIs that should no
-longer be used.
+[_Deprecation_] is "the discouragement of use of some…feature…or practice,
```

Comment:
Is it common practice to omit spaces when using an ellipsis in English? (In German, it is not.)

Reply:
Judging from https://en.wikipedia.org/wiki/Ellipsis#In_American_English, practices vary. I'll add spaces on both sides of each ellipsis.